Cargo.toml

```
@@ -11,10 +11,11 @@ edition = "2018"
 readme = "README.md"
 
 [dependencies]
-embedded-hal = "0.2.2"
+embedded-hal = "0.2"
```

```suggestion
embedded-hal = "0.2.2"
```
Have you checked that this crate would build with `embedded-hal` 0.2.0 and 0.2.1?